electron-app/src/services/types-api.ts

```diff
@@ -57,6 +57,7 @@ export interface ApiManualBalances {
 export interface ApiMakerDAOVault {
   readonly identifier: number;
   readonly name: string;
```

Comment:
Shall we change `name` to `collateral_type` to be in line with the maker docs?
![image](https://user-images.githubusercontent.com/27592/82785892-c27acc80-9e63-11ea-8f6b-1bef4c5d7d12.png)
https://github.com/makerdao/dss/wiki/Glossary

Reply:
It's in the todos of this PR.

Reply:
done.